src/init-ceph.in

```diff
@@ -463,6 +475,7 @@ for name in $what; do
 	    if daemon_is_running $name ceph-$type $id $pid_file; then
 		printf "$name: running "
 		do_cmd "$BINDIR/ceph --admin-daemon $asok version 2>/dev/null" || echo unknown
+		[ `uname` = FreeBSD ] && echo
```

Comment:
@liewegas 
One way or another my Development version strings do not have a '\n' at the end.
And I've been going thru packs of python and other code, but I cannot find.
My linux reference is on CentOS 7 from packages, and that does include a newline in the ceph version output.

And this is a first whack at init-ceph.
It now starts/stops the local deamons, en reports status.
Which is enough IMHO to try and get the package config submitted with FreeBSD ports.